### PR TITLE
#139 Added CameraViewActivity to store Student images and adapted the…

### PR DESCRIPTION
**Pull request by schaemik**
… MultimediaHelper

### CODE REVIEW

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+import java.io.File;
+import java.util.Date;
+
+
+import ch.zhaw.facerecognitionlibrary.Helpers.FileHelper;
+import ch.zhaw.facerecognitionlibrary.Helpers.MatName;
+import ch.zhaw.facerecognitionlibrary.Helpers.MatOperation;
+import ch.zhaw.facerecognitionlibrary.PreProcessor.PreProcessorFactory;
+
+public class CameraViewActivity extends AppCompatActivity implements CameraBridgeViewBase.CvCameraViewListener2 {
```
Comment (jo-elimu) [inline app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:31]: Remember to add the activity to `AndroidManifest.xml`.

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    private int numberOfPictures;
+    private int count;
+    private String deviceId;
+    private String collectionEventId;
+    private StudentImageCollectionEventDao studentImageCollectionEventDao;
+
+
+
+    static {
+        if (!OpenCVLoader.initDebug()) {
```
Comment (jo-elimu) [inline app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:46]: Is this OpenCV initialization time-consuming?

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+        preview = (JavaCameraView) findViewById(R.id.CameraView);
+
+        preview.setCameraIndex(1);
+
+        preview.setVisibility(SurfaceView.VISIBLE);
+        preview.setCvCameraViewListener(this);
+
+        lastTime = new Date().getTime();
+        timerDiff = 100;
```
Comment (jo-elimu) [inline app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:67]: Move these values to static final variables at the top of the file for easier configuration?

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java (hunk 2 above)
Comment (schaemik) [reply app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:46]: I think it is an insignificant amount of time and we need initialization of the OpenCv library at this point.

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java (hunk 1 above)
Comment (schaemik) [reply app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:31]: Alright

File: app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java (hunk 3 above)
Comment (schaemik) [reply app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java:67]: Done

**Pull request merged**